features/docs/formatters/message.feature

```diff
@@ -43,3 +43,28 @@ Feature: Message output formatter
       testRunFinished
       """
 
+  Scenario: it sets "testRunFinished"."success" to false if something failed
+    Given a file named "features/steps.rb" with:
+      """
+      Given('a passed step') {}
+      Given('a failed step') { fail }
+      """
+    When I run `cucumber features/my_feature.feature --format message`
+    Then output should be valid NDJSON
+    And the output should contain:
+    """
+    {"testRunFinished":{"success":false
+    """
+
+  Scenario: it sets "testRunFinished"."success" to true if nothing failed
+    Given a file named "features/steps.rb" with:
+      """
+      Given('a passed step') {}
+      Given('a failed step') { skip_this_scenario }
+      """
+    When I run `cucumber features/my_feature.feature --format message`
+    Then output should be valid NDJSON
+    And the output should contain:
+    """
+    {"testRunFinished":{"success":true
+    """
```

Comment:
newline

Reply:
Where do you want a newline? Wouldn't you an indent for the docstring intead?

Reply:
In the viewing of the PR it says there isn't a newline (At the end of the entire file). Just a common convention